shardingsphere-features/shardingsphere-shadow/shardingsphere-shadow-api/src/main/java/org/apache/shardingsphere/shadow/api/shadow/column/ShadowOperationType.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.shadow.api.shadow.column;
+
+import lombok.NoArgsConstructor;
+
+import java.util.Optional;
+
+/**
+ * Operation types supported by shadow.
+ */
+@NoArgsConstructor
+public enum ShadowOperationType {
+    
+    /**
+     * The shadow operation is insert.
+     */
+    INSERT,
+    
+    /**
+     * The shadow operation is delete.
+     */
+    DELETE,
+    
+    /**
+     * The shadow operation is update.
+     */
+    UPDATE,
+    
+    /**
+     * The shadow operation is select.
+     */
+    SELECT;
+    
+    /**
+     * Create shadow operation type.
+     *
+     * @param operationType operation type
+     * @return shadow operation type
+     */
+    public static Optional<ShadowOperationType> newInstance(final String operationType) {
```

Comment:
Could your refer to `MySQLAuthenticationMethod`?

Reply:
OK.I will optimization method name.